graph/client/src/app/routes.tsx

```diff
@@ -174,4 +210,13 @@ export const releaseRoutes: RouteObject[] = [
       ...childRoutes,
     ],
   },
+  {
+    path: 'project-details/:projectName',
+    id: 'selectedProjectDetails',
+    element: <ProjectDetails />,
+    loader: async ({ request, params }) => {
```

Comment:
suggestion: this route should be feature-flagged in release so that user can't reach it unintentionally. 

You can feature flag the route in the loader by adding a check for the `showExperimentalFeatures` property in the environment:

```typescript
    const environmentConfig = getEnvironmentConfig();

    if (!environmentConfig.appConfig.showExperimentalFeatures) {
        return redirect(`/projects`);
    }
```

Reply:
that would mean we can only look at the view through the dev server and break `nx --show`...
 With the hidden `--web` flag, I think chances are already pretty low of anyone stumbling over it.